amethyst_renderer/src/lib.rs

```diff
@@ -209,7 +216,43 @@ impl Renderer {
     pub fn create_pipe(&mut self, pb: PipelineBuilder) -> Result<Pipeline> {
         pb.build(&mut self.factory, &self.main_target)
     }
-
+	/// Updates the rendering target (useful for resizes)
+	/// Works only on tux for now >_<
+	pub fn regen_target(&mut self)->Option<Target>{
```

Comment:
This function isn't working with backends other than openGL.
For example `gfx_window_dxgi` has no function named `new_views`

Reply:
I'd be curious to know how that code even compiled to be honest.

Reply:
You aways tried it with glutin